perllib/FixMyStreet/Script/Merton/SendWaste.pm

```diff
@@ -104,15 +104,17 @@ sub set_echo_id {
         my $cfg = $cobrand->feature('echo');
         my $echo = Integrations::Echo->new(%$cfg);
         my $event = $echo->GetEvent($problem->external_id);
-        return unless $event && $event->{Id};
-
         my $db = FixMyStreet::DB->schema->storage;
         $db->txn_do(sub {
             my $row2 = FixMyStreet::DB->resultset('Problem')->search({ id => $problem->id }, { for => \'UPDATE' })->single;
-            $row2->update_extra_field({
-                name => 'echo_id',
-                value => $event->{Id},
-            });
+            if ($event && $event->{Id}) {
+                $row2->update_extra_field({
+                    name => 'echo_id',
+                    value => $event->{Id},
+                });
+            } else {
+                $row2->set_extra_metadata( no_echo => 2 );
```

Comment:
Perhaps a comment here to explain what `2` means, for future greppers?

Reply:
Turns out '1' wasn't actually used, that was code that hadn't been removed when Merton wanted garden transfer done entirely differently, so will just switch to more normal one.